sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -1114,6 +1114,56 @@ export class NotificationActionsService {
     );
   }
 
+  /**
+   * Creates Provincial Daily Disbursement Report file processing notifications.
+   * Currently multiple notifications are created as integration contacts
+   * can be multiple.
+   * @param attachmentFileContent file content for the daily disbursement report file.
+   * @param fileName file name for the daily disbursement report file.
+   */
+  async saveProvincialDailyDisbursementReportProcessingNotification(
+    attachmentFileContent: string,
+    fileName: string,
```

Comment:
Can we follow the same approach from the other method and create a `notification` parameter and an interface to t?